Makefile

```diff
@@ -1,15 +1,20 @@
 # Including a Makefile from the visitors package, but making sure
 # to give decent errors.
 
+ifneq ($(shell uname),Darwin)
```

Comment:
hmm, I guess I could clean this up by adding an option like `NO_SHELLSTATUS=1` to only skip this check in opam. open to suggestions